src/SonarQube.Client/Models/SonarQubeDescriptionSection.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarQube.Client.Models
+{
+    public class SonarQubeDescriptionSection
```

Comment:
Don't we already have DescriptionSections?

Reply:
One of them is for parsing the response from the server which is a private class and this one is for the response object. This is for the logical separation of those endpoints.

Reply:
But you also have the one that's used in IRuleDescription, which means you have 3 identical classes, no?

Reply:
IRuleInfo*